frontend/language/src/nb.json

```diff
@@ -1087,7 +1087,7 @@
   "ux_editor.component_properties.dateSent": "Datoinstansen ble sendt inn",
   "ux_editor.component_properties.display": "Visning",
   "ux_editor.component_properties.elements": "Elementer som skal vises (når ingen er satt vises alle)",
-  "ux_editor.component_properties.formatting": "Formattering (formatting)",
+  "ux_editor.component_properties.formatting": "Formatering (formatting)",
```

Comment:
Hadde det vært en fordel for teamet om vi dro inn CSpell? http://cspell.org/ 🤔 Hva tenker du om det? :)

Reply:
Det kan absolutt være verdt å sjekke det ut. Vet du om det støtter norsk? Tok en rask kikk på dokumentasjonen, men kunne ikke finne noe info om det.